AppController/lib/taskqueue.rb

```diff
@@ -37,8 +37,7 @@ module TaskQueue
   # The location of the taskqueue server script. This service controls 
   # and creates celery workers, and receives taskqueue protocol buffers
   # from AppServers.
-  TASKQUEUE_SERVER_SCRIPT = File.dirname(__FILE__) + "/../../AppTaskQueue" + \
-                            "taskqueue_server.py"
+  TASKQUEUE_SERVER_SCRIPT = '/usr/local/bin/appscale-taskqueue'
```

Comment:
It gets created during the build. (`python setup.py install` inside the AppTaskQueue directory). I wanted to do a `which appscale-taskqueue` here, but /usr/local/bin isn't on the path for init scripts unless we wanted to change it explicitly there.